ui/ui-app/src/services/useConfigService.ts

```diff
@@ -136,11 +138,46 @@ export function getRegistryConfig(): ApicurioRegistryConfig {
     return config;
 }
 
-const registryConfig: ApicurioRegistryConfig = getRegistryConfig();
+function overrideObject(base: any, overrides: any | undefined): any {
+    if (overrides === undefined) {
+        return {
+            ...base
+        };
+    }
+    const rval: any = {};
+    Object.getOwnPropertyNames(base).forEach(propertyName => {
```

Comment:
When I thought about it, my first instinct would have been to iterate on the property names coming from `overrides` and just squash into `base` without any validation.
With this implementation, instead, we will be filtering out any "additional"/"unknown" property.

No strong objection to this implementation, I was just thinking that less validation and just adding up whatever comes from the server gives more "freedom" to experiment things out during development.

Reply:
Interesting point, actually.  My current implementation means that any new config MUST be available on the server or it won't be available.  Thinking about it now, that might be a desirable outcome.  It will force developers to follow the "single source of config" principal that this entire feature is based on.  So even though I didn't intend it when I wrote this code, I think I may have accidentally done the right thing here.  :)

Reply:
I do not dislike this thinking at all, but, to make it effective we should probably add some logging about ignored local properties.